rotkehlchen/tests/external_apis/test_gnosispay.py

```diff
@@ -127,3 +130,19 @@ def test_gnosispay_periodic_task(task_manager, database, gnosispay_credentials):
             has_premium=True,
         )
     assert new_events == [event]
+
+
+@pytest.mark.parametrize('function_scope_initialize_mock_rotki_notifier', [True])
+def test_gnosis_pay_unauthorized(database, gnosispay_credentials):
+    """Test if is pop the gnosis pay session expired message"""
+
+    gnosispay = init_gnosis_pay(database)
+
+    _ = gnosispay.get_data_for_transaction(tx_hash='', tx_timestamp=0)
+
+    mocked_notifier = database.msg_aggregator.rotki_notifier
+
+    assert mocked_notifier.pop_message() == MockedWsMessage(
```

Comment:
```suggestion
    assert database.msg_aggregator.rotki_notifier.pop_message() == MockedWsMessage(
```

no need to create a variable if you use it only once